src/averageAges.js

```diff
@@ -20,6 +20,17 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let person = people.filter(man => man.sex === 'm');
+
+  person = (century === undefined)
+    ? person
+    : person.filter(boy => Math.ceil(boy.died / 100) === century);
+
+  const averageSum = person.reduce(
+    (sum, boy) => sum + boy.died - boy.born, 0
+  );
```

Comment:
1. move it into function and reuse in other functions as well
2. use decompose structure
```suggestion
  const averageSum = person.reduce(
    (sum, { died, born }) => sum + (died - born), 0
  );
```